themes/default/content/docs/support/troubleshooting.md

```diff
@@ -406,17 +256,14 @@ const parent = new SomeResource("name", { provider });
 const ids = aws.ec2.getSubnetIds(..., { parent });
 ```
 
-This warning may be benign. However, if you are experiencing crashes or hangs in Pulumi (especially in Node.js version 12.11.0 and
-above) and you see this warning, then it is likely that this is the source.
+This warning may be benign. However, if you are experiencing crashes or hangs in Pulumi (especially in Node.js version 12.11.0 and above) and you see this warning, then it is likely that this is the source.
```

Comment:
```suggestion
This warning may be benign. However, if you are experiencing crashes or hangs in Pulumi (especially in Node.js version 12.11.0 and above) and you see this warning, then it is likely that the condition described above is the root cause.
```

Reply:
@jkodroff we should avoid directional words, because they are not accessible. i can link to the previous section though if you'd like.

Reply:
@susanev This is helpful information. If you have a source handy, can you send me some guidelines on how to refer to previous information that refers to e.g. the preceding paragraph?

The main thing I was trying to address was the use of "this" and "that" with no surrounding context. I _think_ they're called "unqualified articles", and I was taught that they should be avoided in technical writing, but a Google search only turns up how journalists shouldn't write about science, which is obviously not what we're talking about here.

Reply:
we should link to the section we are referencing. i think i put this in the style guide. but maybe im making that up.
some sources that follow this model.
- https://styleguide.mailchimp.com/writing-for-accessibility/#header-3-avoid-directional-language
- https://uxdesign.cc/ux-copy-for-accessibility-2927491cadf8 "→ Avoid using directional language.
Above, below, left, right… Far more often than you might think, these words are meaningless and misinterpreted by the users (and not just those who are blind or visually impaired). Users with screen readers won’t have right or left as a reference."
- https://developers.google.com/style/accessibility "Don't use directional language to orient the reader, such as above, below, or right-hand side. This type of language doesn't work well for accessibility or for localization reasons."

Reply:
ive clarified the language and removed the last reference to `that`. let me know if it makes sense.

Reply:
i think imma go ahead and merge tonight, happy to come back with a separate pr if i got it wrong.